elements/pfe-button/BaseButton.scss

```diff
@@ -0,0 +1,488 @@
+    var(--pf-global--BorderWidth--sm, 1px)) !important;
+  border-radius: var(--pf-c-button--after--BorderRadius,
+    var(--pf-global--BorderRadius--sm, 3px)) !important;
+}
+
+::slotted(button:active) {
+  --pf-c-button--m-primary--Color: var(--pf-c-button--m-primary--active--Color,
+    var(--pf-global--Color--light-100, #fff));
+  --pf-c-button--m-primary--BackgroundColor: var(--pf-c-button--m-primary--active--BackgroundColor,
+    var(--pf-global--primary-color--200, #004080));
+  --pf-c-button--after--BorderWidth: var(--pf-c-button--active--after--BorderWidth,
+    var(--pf-global--BorderWidth--md, 2px));
+
+  // DANGER
+  --pf-c-button--m-danger--Color: var(--pf-c-button--m-danger--active--Color,
+    var(--pf-global--Color--light-100, #fff));
+  --pf-c-button--m-danger--BackgroundColor: var(--pf-c-button--m-danger--active--BackgroundColor,
+    var(--pf-global--danger-color--200, #a30000));
+}
+
+::slotted(button:focus) {
+  --pf-c-button--m-primary--Color: var(--pf-c-button--m-primary--focus--Color,
+    var(--pf-global--Color--light-100, #fff));
+  --pf-c-button--m-primary--BackgroundColor: var(--pf-c-button--m-primary--focus--BackgroundColor,
+    var(--pf-global--primary-color--200, #004080));
+  --pf-c-button--after--BorderWidth: var(--pf-c-button--focus--after--BorderWidth,
+    var(--pf-global--BorderWidth--md, 2px));
+
+  // DANGER
+  --pf-c-button--m-danger--Color: var(--pf-c-button--m-danger--focus--Color,
+    var(--pf-global--Color--light-100, #fff));
+  --pf-c-button--m-danger--BackgroundColor: var(--pf-c-button--m-danger--focus--BackgroundColor,
+    var(--pf-global--danger-color--200, #a30000));
+}
+
+::slotted(button:hover) {
+  --pf-c-button--m-primary--Color: var(--pf-c-button--m-primary--hover--Color,
+    var(--pf-global--Color--light-100, #fff));
+  --pf-c-button--m-primary--BackgroundColor: var(--pf-c-button--m-primary--hover--BackgroundColor,
+    var(--pf-global--primary-color--200, #004080));
+  --pf-c-button--after--BorderWidth: var(--pf-c-button--hover--after--BorderWidth,
+    var(--pf-global--BorderWidth--md, 2px));
+
+  // DANGER
+  --pf-c-button--m-danger--Color: var(--pf-c-button--m-danger--hover--Color,
+    var(--pf-global--Color--light-100, #fff));
+  --pf-c-button--m-danger--BackgroundColor: var(--pf-c-button--m-danger--hover--BackgroundColor,
+    var(--pf-global--danger-color--200, #a30000));
+}
+
+/******************************
+ *                            *
+ *          DANGER            *
+ *                            *
+ ******************************/
+
+:host([variant=link]) ::slotted(button),
+:host([variant=link]) #container {
+  color: var(--pf-c-button--m-link--Color,
+    var(--pf-global--link--Color, #06c)) !important;
+}
+
+:host([variant=link]) ::slotted(button) {
+  background-color: var(--pf-c-button--m-link--BackgroundColor,
+    var(--pf-c-button--m-link--hover--BackgroundColor, transparent)) !important;
+}
+
+:host([danger]) #container ::slotted(button) {
```

Comment:
The #container selector here causes the Secondary danger button to look like a primary, removing it seems to solve the issue.

```Suggestion
:host([danger]) ::slotted(button)
```